src/test/munit/sftp-write-test-case.xml

```diff
@@ -68,4 +68,52 @@
         </munit:validation>
     </munit:test>
 
+    <munit:test name="sftp-write-on-invalid-working-directory" expectedErrorType="SFTP:ILLEGAL_PATH">
+        <munit:behavior>
+            <set-variable variableName="fileName" value="filePath.txt"/>
+        </munit:behavior>
+        <munit:execution>
+            <sftp:write config-ref="config-invalid-folder" path='#[vars.fileName]' createParentDirectories="false">
+                <sftp:content>
```

Comment:
This can be shortened to: <sftp:content>Any string</sftp:content>

Reply:
Fixed